src/cli/commands/dev.ts

```diff
@@ -60,7 +57,7 @@ export class Dev implements Command {
     }
 
     await createWatcher({
-      inspectBrk: args["--inspect-brk"],
+      inspectBrk: args['--inspect-brk'],
       plugins: [layoutPlugin].concat(plugins.map((p) => p.hooks)),
       sourceRoot: layout.sourceRoot,
     })
```

Comment:
@Weakky could you adjust your VSCode settings so autoformat happens in a more guaranteed way? If you're against happy to look into some CI support for it.

Reply:
Thought it was on, I updated my settings let's see if it works! 👍